### ISSUE TITLE
Could not resolve dependency: npm ERR! peer react@"^16.5.2" from react-to-pdf@0.0.13

### ISSUE THREAD

**Issue by dharshan0011** [ivmarcos/react-to-pdf#50]

**Comment by digidub** [comment]
I am also experiencing this issue.

`npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: cv-project@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.5.2" from react-to-pdf@0.0.13
npm ERR! node_modules/react-to-pdf
npm ERR!   react-to-pdf@"*" from the root project`

**Comment by Givo29** [comment]
What version of react are you guys using?

Maybe try using the newest version of react-to-pdf (`0.0.14`)

**Comment by Smartmikey** [comment]
I get the same error while trying to install the Version 0.0.14 of react to pdf

**Comment by Givo29** [comment]
@Smartmikey what version of react are you using?

**Comment by Sjark** [comment]
I'm having the same problem, using React 17.0.1

**Comment by Smartmikey** [comment]
@Givo29 I'm using React version 17.0.1

**Comment by afrancht** [comment]
Bump on this!

**Comment by guerillarealty** [comment]
I'm having the same issue while working with React 17.0.2

**Comment by singhkunal2050** [comment]
Same for me with React 18.0.0

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: procvbuilder@0.1.0
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"^18.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.5.2" from react-to-pdf@0.0.14
npm ERR! node_modules/react-to-pdf
npm ERR!   react-to-pdf@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\UP1479\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\UP1479\AppData\Local\npm-cache\_logs\2022-04-18T04_27_36_415Z-debug.log


```

**Comment by Cwittofur** [comment]
The fix for this issue would be to update the package.json file for this project itself. 

The workaround would be to `npm i --legacy-peer-deps` which will cause npm to install this and its peer dependencies (in this case React 16.5.2) it's not ideal, but it works.

**Comment by ivmarcos** [comment]
v1 has been released supporting react > 16.8
